README.md

```diff
@@ -166,7 +166,7 @@ kubectl port-forward service/ambassador-admins 8877 -n ambassador
 [open|xdg-open] http://localhost:8877/ambassador/v0/diag
 ```
 
-*Ambassador `Mapping` samples above are [disabled](applications/values.yaml) by default because the recommended way is to use host-based routing which requires a domain*
```

Comment:
`recommended way is to use host-based routing which requires a domain` what is this about?
@niqdev

Reply:
@conorfennell see #20